### PR TITLE
Draft smapr blog post

### PR DESCRIPTION
**Pull request by mbjoseph**
Hiya @stefaniebutland  - here's a draft blog post for smapr (feat. rnoaa too!). Any feedback would be appreciated!

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Thanks @mbjoseph. Catchy title! and the preview looks good. I'll get you feedback before the end of day 2018-09-18.

File: content/blog/2018-09-14-smapr.md [hunk 1]
```diff
@@ -0,0 +1,637 @@
+---
+slug: "smapr"
+title: Mapping the 2018 East Africa floods from space with smapr
+package_version: 0.1.2
+authors:
+  - name: Max Joseph
+    url: https://github.com/mbjoseph
+date: 2018-09-14
+categories: blog
+topicid: 925
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: leave `topicid` blank - I'll add ID just before publishing

File: content/blog/2018-09-14-smapr.md [hunk 2]
```diff
@@ -0,0 +1,637 @@
+---
+slug: "smapr"
+title: Mapping the 2018 East Africa floods from space with smapr
+package_version: 0.1.2
+authors:
+  - name: Max Joseph
+    url: https://github.com/mbjoseph
+date: 2018-09-14
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: change date to 2018-09-25

File: content/blog/2018-09-14-smapr.md [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+- software
+- review
+- onboarding
+- package
+- data-access
+- geospatial
+---
+
+Hundreds of thousands of people in east Africa have been displaced and hundreds have died as a result of torrential rains which ended a drought but saturated soils and engorged rivers, resulting in extreme flooding in 2018.
+This post will explore these events using the R package smapr, which provides access to global satellite-derived soil moisture data collected by the NASA Soil Moisture Active-Passive (SMAP) mission and abstracts away some of the complexity associated with finding, acquiring, and working with the HDF5 files that contain the observations (shout out to [Laura DeCicco](https://github.com/ldecicco-USGS) and [Marco Sciaini](https://github.com/marcosci) for reviewing smapr, and [Noam Ross](https://github.com/noamross) for editing in the Ropensci onboarding process).
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: s/Ropensci/rOpenSci/ & check for proper capitalization in all cases

File: content/blog/2018-09-14-smapr.md [hunk 4]
```diff
@@ -0,0 +1,637 @@
...
+We will focus on Somalia and Kenya, two of the hardest hit countries.
+We'll also lean on another Ropensci package, rnoaa, to link precipitation to soil moisture.
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: put names of all packages, including smapr, in backticks, like `smapr`, `rnoaa`

File: content/blog/2018-09-14-smapr.md (hunk 3 above)
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: can you link "NASA Soil Moisture Active-Passive (SMAP) mission" somewhere so readers can get more info if interested?

Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: shoutouts to reviewers and editors always appreciated!

File: content/blog/2018-09-14-smapr.md [hunk 5]
```diff
@@ -0,0 +1,637 @@
...
+- community
+- software
+- review
+- onboarding
+- package
+- data-access
+- geospatial
+---
+
+Hundreds of thousands of people in east Africa have been displaced and hundreds have died as a result of torrential rains which ended a drought but saturated soils and engorged rivers, resulting in extreme flooding in 2018.
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: This is an excellent intro - makes me want to read on and frames the issue for which the packages are developed & used

File: content/blog/2018-09-14-smapr.md [hunk 6]
```diff
@@ -0,0 +1,637 @@
...
+## 1 /home/rstudio/.cache/smap
+## 2 /home/rstudio/.cache/smap
+## 3 /home/rstudio/.cache/smap
+## 4 /home/rstudio/.cache/smap
+## 5 /home/rstudio/.cache/smap
+## 6 /home/rstudio/.cache/smap
+```
+
+Next, we can list the contents of the files we downloaded.
+These are HDF5 files, so each file contains multiple datasets.
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: Can you link HDF5 for people unfamiliar to get more info? e.g. https://portal.hdfgroup.org/pages/viewpage.action?pageId=48804287 or https://en.wikipedia.org/wiki/Hierarchical_Data_Format

File: content/blog/2018-09-14-smapr.md [hunk 7]
```diff
@@ -0,0 +1,637 @@
...
+  mean_sm
+}
+```
+
+Notice that spatial coverage is still not 100%.
+Notably, places where the satellite missed and frozen regions in the arctic have `NA` values.
+
+## Getting global precipitation data with rnoaa
+
+We can better interpret soil moisture if we know about precipitation, and the NOAA Climate Prediction Center (CPC) has global precipitation data that are a cinch to get with the `cpc_prcp` function.
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: reading to this point and your narrative style is fantastic - good clear explanations of what you're doing and what is represented in the figs

File: content/blog/2018-09-14-smapr.md [hunk 8]
```diff
@@ -0,0 +1,637 @@
...
+
+![](/img/blog-images/2018-09-14-smapr/joint-timeseries.gif)
+
+Here the soil moisture and precipitation values are mapped over time, with line plots below to show the median value (black line), interquartile range (dark grey ribbon), 10% and 90% quantiles (light grey ribbon).
+The exceptional nature of the flooding in 2018 shows up as a high and broad peak in soil moisture and precipitation, and the drought that preceded the flooding in 2017 is visible as a period of low rains and dry soil.
+
+
+### Want to contribute?
+
+If you're psyched on global soil moisture and want to contribute to smapr, there is room to develop support for more level 2 products (currently smapr supports the more processed level 3 science grade and level 4 enhanced value products primarily).
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: It is GREAT that you identify how people can contribute!

Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: For a novice reader (like me), can you indicate what level 2 vs level 3, 4 products are?

File: content/blog/2018-09-14-smapr.md [hunk 9]
```diff
@@ -0,0 +1,637 @@
...
+```
+## Joining, by = c("x", "y", "date")
+```
+
+![](/img/blog-images/2018-09-14-smapr/pr-vs-sm-1.png)<!-- -->
+
+Notice the band of zero-precipitation points that sit on the y-axis.
+Among nonzero precipitation values, it seems like there is a nonlinear relationship between precipitation and soil moisture.
+
+So, now let's take a look at the spring and summer of 2018, when the flooding was worst in Somalia and Kenya.
```
Comment (stefaniebutland) [inline content/blog/2018-09-14-smapr.md]: minor comment: with the paired plots that follow where you have first soil moisture, then precipitation, it took me a bit to see that the plots were paired and the fig legend told me what the plot was. If it doesn't change the flow much, perhaps you could add "Soil moisture" or "Precipitation" in regular text above each.

File: content/blog/2018-09-14-smapr.md (hunk 5 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Thanks @stefaniebutland! Glad you like it.

File: content/blog/2018-09-14-smapr.md (hunk 9 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Done!

File: content/blog/2018-09-14-smapr.md (hunk 1 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Done! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

File: content/blog/2018-09-14-smapr.md (hunk 2 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Done! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

File: content/blog/2018-09-14-smapr.md (hunk 3 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Good catch! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

File: content/blog/2018-09-14-smapr.md (hunk 4 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Got it! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

File: content/blog/2018-09-14-smapr.md (hunk 3 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Yep - done here: https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Their work was appreciated!

File: content/blog/2018-09-14-smapr.md (hunk 6 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Yep! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

File: content/blog/2018-09-14-smapr.md (hunk 7 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Thanks!

File: content/blog/2018-09-14-smapr.md (hunk 8 above)
Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: :+1:

Comment (mbjoseph) [reply content/blog/2018-09-14-smapr.md]: Done! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1

**Comment by mbjoseph** [comment]
@stefaniebutland thanks for your review! I'm glad you liked the post, and I appreciate your feedback. Looking forward to getting this out. If there's anything to highlight in a tweet, maybe it's something around the relevance of flooding with hurricane Florence, or something about how `smapr` and `rnoaa` are friends/the rOpenSci package ecosystem increases the accessibility of this kind of analysis?

**Comment by stefaniebutland** [comment]
Thanks for the quick fixes and the tweet tips.
I'm not sure how I would phrase something around the relevance of flooding with hurricane Florence without sounding really naive. Let me know if you have any specific text suggestion (no obligation - all your time spent already is very much appreciated).

**Comment by stefaniebutland** [comment]
note to self: tag https://twitter.com/mxwlj

**Comment by mbjoseph** [comment]
No worries - I think just highlighting the smapr + rnoaa combo is probably fine!

**Pull request merged**